server/games/game_results.py

```diff
@@ -142,6 +144,11 @@ def victory_only_score(self, army: int):
                 startspot, score = row[0], row[1]
                 # FIXME: Assertion about startspot == army
                 # FIXME: Reporter not retained in database
-                result = GameResult(0, startspot, GameOutcome.UNKNOWN, score)
+                # TODO: I don't understand the comment above
```

Comment:
Yea I'm not sure what that comment refers to either. As for the `startspot == army` I think that's because the lua code always assumes that to be true, and I guess therefore the server does too, but it isn't checked for or enforced.